rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -54,7 +54,127 @@ paths:
           $ref: '#/components/responses/NotImplemented'
         default:
           $ref: '#/components/responses/DefaultError'
-
+  /transform/{TransformId}:
+    parameters:
+      - $ref: '#/components/parameters/TransformId'
+    get:
+      summary: Poll transformation request status
+      description: >
+        After posting a transformation request, the client can use the returned URI to poll for the transform's completion at this endpoint.
+      operationId: poll
+      tags:
+        - transform
+      responses:
+        '200':
+          description: The transform is still in progress.
+          headers:
+            Content-Version:
+              $ref: '#/components/headers/ContentVersion'
+        '201':
+          description: Transform complete. Returning metadata locations.
+          headers:
+            Content-Version:
+              $ref: '#/components/headers/ContentVersion'
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
```

Comment:
✏️ I suggest creating this structure containing the format (might want to call it `type` for consistency with transform), location in another structure named something like MetadataInfo.
❗️ I would also add a modified timestamp to this structure so we can identify the timestamp when we generated this metadata type (as opposed to the timestamp of the product from which we generated this metadata type). This is a piece of info that is required to version all metadata types.

At some point, we will want to add our source id as being `Ion Transform` and the version of ourselves used to generate this metadata. This is info @brendan-hofmann and I have been discussing to keep track of so we know later in a different site, whether or not we should re-generate a particular metadata type when our flows are upgraded.